### PR TITLE
Remove ConsumerParameters from examples/

### PR DESCRIPTION
**Pull request by sbenthall**
Addresses #580

### CODE REVIEW

**Comment by mnwhite** [comment]
example_ConsIndShock.py (and its notebook) still has references to Params.

File: examples/ConsumptionSaving/example_ConsGenIncProcessModel.py [hunk 1]
```diff
@@ -7,13 +7,16 @@
     IndShockExplicitPermIncConsumerType,
     IndShockConsumerType,
     PersistentShockConsumerType,
+    init_explicit_perm_inc
```
Comment (MridulS) [inline examples/ConsumptionSaving/example_ConsGenIncProcessModel.py]: init_persistent_shocks

also needs to be imported here.

**Comment by MridulS** [comment]
example_ConsRepAgentModel.ipynb notebook needs to be updated too.

**Comment by mnwhite** [comment]
All of the notebooks need to be updated.  And that dictionary does not need
to be imported there.  The example file runs correctly.

**Comment by MridulS** [comment]
Oh yeah, you are right. With the class initiations there is no need to import the dictionaries.

@sbenthall  I was running tests locally on examples notebooks.

```
pytest --nbval-lax examples/
```
this will take a *lot* of time so batch them.

**Comment by mnwhite** [comment]
Someone else should play with this branch before we merge it.  But it's ready from my perspective.

**Comment by llorracc** [comment]
@MridulS can you do this?

**Pull request merged**